.github/workflows/backend_lint.yml

```diff
@@ -0,0 +1,31 @@
+# This workflow will install Python dependencies, run tests and lint with a variety of Python versions
+# For more information see: https://help.github.com/actions/language-and-framework-guides/using-python-with-github-actions
+
+name: "[CI] Backend / Lint"
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+
+jobs:
+  backend_lint:
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: Set up Python
+      uses: actions/setup-python@v2
+
+    - name: Install dependencies
+      run: |
+        python -m pip install --upgrade pip
+        python -m pip install flake8
+        python -m pip install pipenv
+        # if [ -f Pipfile ]; then pipenv install; fi
```

Comment:
Here, dont run `pipenv install` run `pipenv instal -d` (`-d` for development-dependencies